src/transformers/models/layoutlmv2/modeling_layoutlmv2.py

```diff
@@ -770,25 +770,27 @@ def _calc_img_embeddings(self, image, bbox, position_ids):
         return embeddings
 
     def _calc_visual_bbox(self, image_feature_pool_shape, bbox, device, final_shape):
-        visual_bbox_x = (
+        visual_bbox_x = torch.div(
             torch.arange(
                 0,
                 1000 * (image_feature_pool_shape[1] + 1),
                 1000,
                 device=device,
                 dtype=bbox.dtype,
-            )
-            // self.config.image_feature_pool_shape[1]
+            ),
+            self.config.image_feature_pool_shape[1],
+            rounding_mode="floor",
```

Comment:
```suggestion
            rounding_mode="trunc",
```
(nit) Shouldn't this be "trunc" rather than "floor"?